projects/packages/jetpack-mu-wpcom/tests/php/features/launchpad/class-launchpad-task-lists-test.php

```diff
@@ -22,4 +22,107 @@ public function test_build_creates_no_PHP_warnings() {
 		$this->assertIsArray( $result );
 		$this->assertEmpty( $result );
 	}
+
+	/**
+	 * Data provider for ::test_task_list_is_completed().
```

Comment:
You should be able to use an inline `@see` here to make this link more explicit.
```suggestion
	 * Data provider for {@see test_task_list_is_completed()}.
```